core/src/main/java/com/exadel/etoolbox/backpack/core/services/pckg/impl/BasePackageServiceImpl.java

```diff
@@ -171,6 +176,10 @@ public PackageInfo getPackageInfo(final ResourceResolver resourceResolver, final
         packageInfo.setToggle(packageModel.isToggle());
         packageInfo.setDataSize(actualPaths.stream().mapToLong(value -> getAssetSize(resourceResolver, value)).sum());
 
+        if (!brokenPaths.isEmpty()) {
+            packageInfo.setResultStatus(new ResultStatus.WarningResultStatus(brokenPaths));
```

Comment:
Yes, makes sense to make this shorter

Reply:
Regarding the successful status, it is taken from the properties <successresponse /> by default, it can be duplicated here for the clarity of the developers, the only question is is it worth it?